utils/tolerance.stanza

```diff
@@ -0,0 +1,60 @@
+#use-added-syntax(jitx)
+defpackage ocdb/tolerance:
+  import core
+  import collections
+  import math
+
+; A generic value with some tolerance. May be offcenter (upper tolerance != lower tolerance)
+public pcb-struct ocdb/tolerance/Toleranced:
+  typ: Double,
```

Comment:
could it be 0.0?

Reply:
It's very unlikely. But we also can't use `with: (ensure)` in pcb-structs.